Classes/AbstractDataHandlerListener.php

```diff
@@ -138,6 +171,142 @@ protected function getAllChangeSetValuesMatch($triggerConfiguration, $changedFie
     }
 
     /**
+     * Retrieves the name of the  Indexing Queue Configuration for a record assumes
```

Comment:
Retrieves the name of the Index Queue Configuration for a record. 
Assumes that $this->solrConfiguration has been initialized properly.